bin/delete_empty_api_groups.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+#
+# Fix the API groups that were automatically created from their singleton configs but
+# hold no actual useful configuration. These API groups block upgrading to Open Forms
+# 3.0 otherwise, unless manually removed in the admin interface.
+#
+
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+
+import django
+from django.db.models import Q
+
+SRC_DIR = Path(__file__).parent.parent / "src"
+sys.path.insert(0, str(SRC_DIR.resolve()))
+
+
+def fix_api_groups():
+    from openforms.contrib.objects_api.models import ObjectsAPIGroupConfig
+    from openforms.registrations.contrib.zgw_apis.models import ZGWApiGroupConfig
+
+    objects_api_groups = ObjectsAPIGroupConfig.objects.filter(
+        objects_service__isnull=True,
+        objecttypes_service__isnull=True,
+        drc_service__isnull=True,
+        catalogi_service__isnull=True,
+    ).only("pk")
+    if objects_api_groups:
```

Comment:
Why do we put the delete outside the if statement?

Reply:
I used it as a sanity check that the `.only` doesn't mess with the `.delete` :) but other than that, no good reason